dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilderTest.java

```diff
@@ -131,15 +130,15 @@ public void testResolveClassPath_optionalDependency() throws RepositoryException
   }
 
   @Test
-  public void testResolveClassPath_invalidCoordinate() {
+  public void testResolveClassPath_invalidCoordinate() throws RepositoryException {
     Artifact nonExistentArtifact = new DefaultArtifact("io.grpc:nosuchartifact:1.2.3");
-    try {
-      classPathBuilder.resolve(ImmutableList.of(nonExistentArtifact));
-      Assert.fail("Invalid Maven coodinate should raise RepositoryException");
```

Comment:
With this PR, missing artifact is recorded as `getArtifactProblem()`, not as an exception.